### PR TITLE
4_carisman

### PR DESCRIPTION
**Pull request by carisno**

### CODE REVIEW

**Review by oddcommits — CHANGES_REQUESTED** [review]
- write about yourself in `carisno/README`
- add you to the contributor list in root README.md

**Comment by carisno** [comment]
I'm carisno. I have resolved mistakes. Please merge.

File: README.md [hunk 1]
```diff
@@ -85,7 +85,8 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 20. The admin will review the PR and say **Congrats** if everything looks good.
 
 ## Contributors
-
 - [John Doe](./john-doe/README.md)
+- [carisno](./carisno/README.md)
```
Comment (oddcommits) [inline README.md:89]: your name should be after `seb` or `whitecollin`

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -85,7 +85,8 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 20. The admin will review the PR and say **Congrats** if everything looks good.
 
 ## Contributors
-
 - [John Doe](./john-doe/README.md)
+- [Carisno Remes](./carisno/README.md)
```
Comment (oddcommits) [inline README.md:89]: ???

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (carisno) [reply README.md:89]: My name is Carisno Remes. So I update carisno to Carisno Remes.

Comment (oddcommits) [reply README.md:89]: you misunderstood. Sebastian and White already merged PR before you

Comment (carisno) [reply README.md]: I see, have fixed this mistake. Sorry.

Comment (carisno) [reply README.md]: > ???
I understood that modify my name.

File: README.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 20. The admin will review the PR and say **Congrats** if everything looks good.
 
 ## Contributors
-
```
Comment (oddcommits) [inline README.md:88]: why removed this line?

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (carisno) [reply README.md]: > I made a mistake. I fixed this.

Comment (oddcommits) [reply README.md]: generally, conversations aren't resolved automatically but manually. but you can see `Outdated` badge if the mentioned code part had been changed

**Comment by oddcommits** [comment]
resolve conflicts please, @carisno

**Review by oddcommits — APPROVED** [review]

**Pull request merged**